docs/static-assets.md

```diff
@@ -113,9 +113,14 @@ Additionally, Vue assumes relative paths. If you are having an Issue with your a
 2. Run:
 
    ```bash
-   ./dark-cli-apple --canvas myusername-mycanvas --password mypass --user myusername build
+    # The single quotes - around mypass help protect against shell escaping and interpolation.
```

Comment:
Do you want to lift this out of the bash code section just above/below? Unsure how this'll render.

Reply:
I put it there 'cuz I'm 70% sure it'll render as a bash comment and I'm not sure how to deal with empty lines in an <ol>. 😂